addons/api/mirage/factories/host-set.js

```diff
@@ -35,21 +35,24 @@ export default factory.extend({
       return faker.datatype.number();
     }
   },
-  // AWS specific
-  filters() {
+
+  attributes() {
+    // AWS specific
     if (this.plugin?.name === 'aws') {
```

Comment:
The azure plugin will have the filter in just `filter` and not `attributes.filters`?

Reply:
Yup. Azure filter supports a single string value whereas aws supports multiple filters as an array.